src/auro-calendar-cell.js

```diff
@@ -299,6 +299,10 @@ export class AuroCalendarCell extends LitElement {
 
   firstUpdated() {
     this.datepicker = this.runtimeUtils.closestElement('auro-datepicker', this);
+
+    this.datepicker.addEventListener('auroDatePicker-toggled', () => {
+      this.handleSlotContent();
+    });
```

Comment:
Do we need to do this? What is the case for doing when the view is toggled rather than just when the content changes?

Reply:
For some reason, there is a loading issue when you use the `@slotchange` event on the `slot` html. This is why we also call `this.handleSlotContent()` on updated as well, rather than the `@slotchange` event handling everything.